benches/benches.rs

```diff
@@ -0,0 +1,227 @@
+#[macro_use]
+extern crate bencher;
+extern crate nalgebra;
+
+
+mod collapse {
+    use bencher::Bencher;
+    use wfc_rust::wfc::collapse::collapse;
+    use wfc_rust::io::text_parser::{parse, make_edges_cardinal_grid};
+    use wfc_rust::graph::graph::{Graph, Rules};
+    use wfc_rust::multiset::{Multiset, MultisetTrait};
+    use wfc_rust::utils::hash_map;
+    use nalgebra::{U6};
+
+    pub fn bench_collapse(bench: &mut Bencher) {
+        let out_width = 100;
+        let out_depth = 100;
+
+        if let Ok((input_graph, _)) = parse::<U6>("resources/test/tosashimizu_model.txt") {
```

Comment:
Is specify `U6` type just for the purposes of benchmarking?

Reply:
That's correct, just using U6 here to benchmark with it.